src/Web/Spock/Internal/Wire.hs

```diff
@@ -54,6 +54,17 @@ import qualified Network.Wai.Parse as P
 instance Hashable StdMethod where
     hashWithSalt = hashUsing fromEnum
 
+data SpockMethod
```

Comment:
I think prefixing the constructors would be good: MethodStandard and MethodCustom e.g.?